The Language Service could not be started.

```
2018-6-11 11:31:48 [NORMAL] - Visual Studio Code v1.24.0 64-bit
2018-6-11 11:31:48 [NORMAL] - PowerShell Extension v1.7.0
2018-6-11 11:31:48 [NORMAL] - Operating System: Windows 64-bit
2018-6-11 11:31:48 [NORMAL] - Path specified by 'powerShellExePath' setting - '' - not found, reverting to default PowerShell path.
2018-6-11 11:31:48 [NORMAL] - Language server starting --
2018-6-11 11:31:48 [NORMAL] -     exe: C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe
2018-6-11 11:31:48 [NORMAL] -     args: C:\Users\rdattilo\.vscode\extensions\ms-vscode.powershell-1.7.0\modules\PowerShellEditorServices\Start-EditorServices.ps1 -HostName 'Visual Studio Code Host' -HostProfileId 'Microsoft.VSCode' -HostVersion '1.7.0' -AdditionalModules @('PowerShellEditorServices.VSCode') -BundledModulesPath 'C:\Users\rdattilo\.vscode\extensions\ms-vscode.powershell-1.7.0\modules' -EnableConsoleRepl -LogLevel 'Verbose' -LogPath 'C:\Users\rdattilo\.vscode\extensions\ms-vscode.powershell-1.7.0\logs\1528731108-8debbaed-db66-4139-b259-39746061c6391528731105860\EditorServices.log' -SessionDetailsPath 'C:\Users\rdattilo\.vscode\extensions\ms-vscode.powershell-1.7.0\sessions\PSES-VSCode-17688-512792' -FeatureFlags @()
2018-6-11 11:31:49 [NORMAL] - powershell.exe started, pid: 19508
2018-6-11 11:32:48 [NORMAL] - Language server startup failed.
2018-6-11 11:32:48 [ERROR] - The language service could not be started: 
2018-6-11 11:32:48 [ERROR] - Timed out waiting for session file to appear.
```

This is all of the sudden started happening.  It was fine a few days ago and then,this :(

I've uninstalled VsCode, uninstalled the powershell add-in..... several times.  

```
Name                           Value
----                           -----
PSVersion                      5.1.17134.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17134.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1`
```

thats my PS version I'm running.  The only thing of late that has changed on my PC is an update to Windows 1803.  Although I was using VSCode just fine after the update.  Maybe there is some sort of weird connection to this?

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -85,8 +85,8 @@ export function waitForSessionFile(sessionFilePath: string, callback: IWaitForSe
         }
     }
 
-    // Try once every 2 seconds for 120 seconds, two full minutes
-    innerTryFunc(120, 2000);
+    // Try once every 2 seconds, 60 times - making two full minutes
+    innerTryFunc(60, 2000);
 }
 
 export function readSessionFile(sessionFilePath: string): IEditorServicesSessionDetails {

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -85,8 +85,8 @@ export function waitForSessionFile(sessionFilePath: string, callback: IWaitForSe
         }
     }
 
-    // Try once per second for 60 seconds, one full minute
-    innerTryFunc(60, 1000);
+    // Try once every 2 seconds for 120 seconds, two full minutes
+    innerTryFunc(120, 2000);
 }
 
 export function readSessionFile(sessionFilePath: string): IEditorServicesSessionDetails {

